src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -18,524 +18,523 @@
 using NuGet.Versioning;
 using Strings = Microsoft.DotNet.Workloads.Workload.Install.LocalizableStrings;
 
-namespace Microsoft.DotNet.Workloads.Workload
+namespace Microsoft.DotNet.Cli;
+
```

Comment:
This file lives in a weird place in the `dotnet` project. I'll likely adjust this in a future update.